src/main/java/com/group4/vms/model/Entry.java

```diff
@@ -13,4 +15,12 @@ public class Entry {
     private String name;
     private ObjectId[] events;
     private double[] timeVolunteered;
+    public List<Double> getTimeVolunteered() {
+        List<Double> timeVolunteeredList = new ArrayList<>();
+        for (double time : timeVolunteered) {
+            timeVolunteeredList.add(time);
+        }
+        return timeVolunteeredList;
+        
+    }
```

Comment:
What is the purpose of this? 
It seems that when this `Document` is instantiated, this getter method will be built into a _model_ class. 
`@Data` give us getters and setters so we don't need to make our own.